lambda/voting/scripts/create.mjs

```diff
@@ -7,7 +7,7 @@ import getDeveloperWallet, {
   getTransactionStatus,
   sleep,
   checkAPIKey,
-} from "./utils.mjs";
+} from "../../../helpers/utils.mjs";
```

Comment:
update transaction check logic in this file to use the utils

Reply:
done